src/main/kotlin/com/manimdsl/runtime/ExecValue.kt

```diff
@@ -59,11 +59,15 @@ sealed class ExecValue {
 
 sealed class PrimitiveValue : ExecValue()
 
-sealed class DoubleAlias() : PrimitiveValue() {
+sealed class DoubleAlias : PrimitiveValue() {
     abstract fun toDouble(): Double
 }
 
 data class DoubleValue(override val value: Double, override var manimObject: MObject = EmptyMObject) : DoubleAlias() {
+
+    override val name: String
+        get() = "Double"
```

Comment:
Can this just be override val name: String = "Double", same for the other Values